english/coffee/nude-run/cold-water/run-away/run-away.md

```diff
@@ -5,4 +5,6 @@ end, it doesn't even matter. You had to fall
 to lose it all, but in the end, it doesn't even
 matter.
 
-Because that badger s-o-b outruns you and kills you.
\ No newline at end of file
+Because that badger s-o-b outruns you and kills you.
+
+["Go to heaven"](../cold-water/run-away/heaven/heaven.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.